contribs/gmf/src/directives/partials/layertree.html

```diff
@@ -1,10 +1,19 @@
 <span ng-if="::!layertreeCtrl.isRoot"
     ng-class="['depth-' + layertreeCtrl.depth, gmfLayertreeCtrl.getResolutionStyle(layertreeCtrl.node), gmfLayertreeCtrl.getNoSourceStyle(layertreeCtrl)]">
   <a href ng-click="::gmfLayertreeCtrl.toggleActive(layertreeCtrl)">
+    <span class="layerIcon">
+      <img ng-src="{{gmfLayertreeCtrl.getLegendIconUrl(layertreeCtrl)}}"></img>
```

Comment:
So it does a request per layer each time the resolution change.
It is a lot of requests.